plugin.go

```diff
@@ -63,6 +64,8 @@ func (p Plugin) Exec() error {
 		secretFile,
 		"--env-file",
 		envFile,
+		"--actor",
```

Comment:
only set this if actor is set

Reply:
https://github.com/drone-plugins/github-actions/pull/9/commits/b18291d15ada2cc232d29939f0f7c526e4750ea9